schemas/geodesyml/0.6/monumentInfo.xsd

```diff
@@ -262,7 +262,8 @@ Copyright: Scripps Orbit and Permanent Array Center (SOPAC), Commonwealth Govern
                     <!-- TODO: use gco -->
                     <element minOccurs="0" name="faultZonesNearby" type="gml:CodeType"/>
                     <element minOccurs="0" name="distance-Activity" type="string"/>
-                    <element minOccurs="0" name="notes" type="string"/>
+                    <!-- <element minOccurs="0" name="notes" type="string"/> -->
+                    <group ref="geo:RemarksGroup"/>
```

Comment:
If element `notes` is not needed anymore (since it is included as part of `geo:RemarksGroup`), then it would be better to remove the commented code. The usual guideline when commenting out code is to either delete it altogether or to also include a comment explaining why the code has been commented out.